### DISCUSSION TITLE
articles/awareness-2023-review

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#185] [Comments]
# articles/awareness-2023-review

2023年过得有点太快，似乎还没有反应过来，就到了2024年。回望这一年，与其说收获，不如说是改变。这一年心态和认知上都发生了改变，我从抑郁的边缘走出来，面对生活与工作有了更多的淡定与从容。

https://hutusi.com/articles/awareness-2023-review

**Comment by xiepeng21** [comment]
“人生更像是在一个黑暗的森林里寻路，摆在面前的永远是多种未知的道路选择，而随着阅历和经验的增加，我们会形成自己的判断逻辑，未来永远是未知和不确定的，我们对世界的认识永远是困惑的。而对于不确定的未来，我希望在认知和方法论、AI技术、人文三个方向持续学习，去探知这个世界。”，讲得很好。另外文中的那张图，回看来时路，展望未来路，一图胜千言，很棒，也是对本文最好的总结。

**Reply by hutusi** [reply]
谢谢~

**Comment by wangyunzi** [comment]
最近在看susan写的《why grow up》这本书，希望也推荐给你，虽然看起来和你现在的境遇不太相符，但是我认为这是一本所有人在自己人生转折点时刻都值得一看的书，good luck to you～！

**Reply by hutusi** [reply]
谢谢~这本书看了下介绍不错，已经纳入reading list😀

**Reply by hutusi** [reply]
春节期间看完了这本书，非常棒！谢谢推荐👍

**Comment by DemoJ** [comment]
感谢。。以前看《纳瓦尔宝典》，都着眼于前两章了，没想到「幸福」篇写的这么好，很有启发意义，谢谢博主

**Reply by hutusi** [reply]
全书信息量浓缩度很高~~

**Comment by jtsang4** [comment]
难得看到这么坦诚清晰，平静却有力量的文章，剖析自己就是自我提升的方式。

> 他在二十岁左右就开始认知觉醒，开始去探究这个世界运行的本质，因而在每日的工作和学习中持续摸索出了自己的方法论和系统化思维方式，这也是他的进步之道。

看到这句想到了自己也仍在慢慢摸索呀，共勉～

**Reply by hutusi** [reply]
谢谢，共勉：）

**Comment by petite-dd** [comment]
从心所欲不逾矩，这我仔细思索过，儒家是入世的，入市且从心所欲且不逾矩，这是一种至高境界。

**Comment by 4mengy** [comment]
“跳到直面生死的层次去思考”，又如何避免进入虚无主义的陷阱呢？

**Reply by hutusi** [reply]
知行合一。想到了就去做，做具体的事情。
